react/src/components/MapProjectNavBar/MapProjectNavBar.tsx

```diff
@@ -96,6 +104,9 @@ const MapProjectNavBar: React.FC<NavBarPanelProps> = ({ isPublicView }) => {
           if (activePanel === item.panel) {
             // If already active, we want to remove queryPanel key if user clicks again
             updatedQueryParams.delete(queryPanelKey);
+          }
+          if (!isPublicView && item.showWhenPrivate == false) {
```

Comment:
could this be moved to next to line 100 `if (isPublicView && !item.showWhenPublic)` or maybe combined with it?

Reply:
could the useEffect on line 79-90 be updated too to remove the query param for the `!isPublicView && item.showWhenPrivate == false` case.